pkgs/applications/misc/haxor-news/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, pythonPackages }:
+
+pythonPackages.buildPythonApplication rec {
+  version = "0.3.1";
+  name = "haxor-news-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/donnemartin/haxor-news/archive/0.3.1.tar.gz";
+    sha256 = "0jglx8fy38sjyszvvg7mvmyk66l53kyq4i09hmgdz7hb1hrm9m2m";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    click
+    colorama
+    requests2
+    pygments
+    prompt_toolkit_52
+    six
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = src.url;
```

Comment:
could you just put the actual url here. 
You would actually need to inherit src to make this work. Travis now failed because of this.